src/views/Dashboard/addAccommodationPage.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import Styles from "../../styles/entry.module.css";
+import styles2 from "../../styles/addAccom.module.css";
+import AddAccommodationForm from "../../components/AddAccommodationForm";
+import { addAccommodation } from "../../redux/actions/addAccommodationAction";
+
+class addAccommodationPage extends Component {
+  submit = (data) =>
+    this.props
+      .addAccommodation(data)
+      .then(() => this.props.history.push("/dashboard"));
```

Comment:
You need to handle any error that might happen in the backend side

Reply:
Errors from the backend are handled but not in this particular file.